README.md

```diff
@@ -87,13 +87,13 @@ The next two sections presume you are using the out-of-the-box `.mf.config.json`
 
 # Firebolt SDK Support
 
-Mock Firebolt supports these Firebolt SDKs: **core**, **manage**, **discovery**.
+Mock Firebolt supports these Firebolt SDKs: **core**, **discovery**.
```

Comment:
See earlier comment. From an SDK perspective, we still do support the manage SDK.